core/data/src/test/java/uk/gov/gchq/gaffer/data/element/function/ReduceRelatedElementsTest.java

```diff
@@ -378,7 +378,7 @@ protected ReduceRelatedElements getInstance() {
     }
 
     @Override
-    protected Iterable getDifferentInstancesOrNull() {
+    protected Iterable<ReduceRelatedElements> getDifferentInstancesOrNull() {
```

Comment:
As part of parameterising all of the `FunctionTest`s, function signatures like this were corrected